src/TestFramework/SafeQuery.php

```diff
@@ -33,12 +33,28 @@
 
 declare(strict_types=1);
 
-namespace Infection\Config\Guesser;
+namespace Infection\TestFramework;
+
+use DOMElement;
+use DOMNodeList;
+use DOMXPath;
+use Webmozart\Assert\Assert;
 
 /**
  * @internal
  */
-interface Guesser
+trait SafeQuery
 {
-    public function guess();
+    /**
+     * @return DOMNodeList|DOMElement[]
+     * @phpstan-return DOMNodeList<DOMElement>
+     */
+    private static function safeQuery(DOMXPath $xPath, string $query): DOMNodeList
```

Comment:
What do you think about making it a static helper instead of a trait?

Reply:
For now i'm happy to leave this as a trait.